osmaxx/excerptexport/migrations/0025_move_bounding_geometry_20160503_1639.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-05-03 14:39
+from __future__ import unicode_literals
+
+from django.contrib.gis.geos import Polygon, MultiPolygon
+from django.db import migrations
+
+
+def multi_polygon_from(bounding_geometry_old):
+    bb = bounding_geometry_old
+    west, south, east, north = bb.south_west[0], bb.south_west[1], bb.north_east[0], bb.north_east[1]
+    polygon = Polygon.from_bbox((west, south, east, north))
+    return MultiPolygon(polygon)
+
+
+def move_bounding_geometry_data_to_new_multipolygon_field(apps, schema_editor):
+    Excerpt = apps.get_model('excerptexport', 'Excerpt')  # noqa
+    BBoxBoundingGeometry = apps.get_model('excerptexport', 'BBoxBoundingGeometry')  # noqa
+    for excerpt in Excerpt.objects.all():
+        excerpt.bounding_geometry = multi_polygon_from(
+            BBoxBoundingGeometry.objects.get(pk=excerpt.bounding_geometry_old.id)
```

Comment:
Yes, it should, but I wasn't sure if the in Memory-Model does that correctly, and instead of a test I just used the safest way, which will fail as loud as possible if something is off..